src/panel/poi/blocks/Details.jsx

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
+import StarsBlock from './Stars';
 import AccessibilityBlock from './Accessibility';
-// import BreweryBlock from './Brewery';
 import InternetAccessBlock from './InternetAccess';
 import DeliveryBlock, { hasActiveDeliveryModes } from './Delivery';
 import { Divider } from 'src/components/ui';
 import { findBlock } from 'src/libs/pois';
 import { useI18n } from 'src/hooks';
+import poiSubClass from 'src/mapbox/poi_subclass';
 
 const DetailsBlock = ({ poi }) => {
   const { _ } = useI18n();
-
+  const subclass = poiSubClass(poi.subClassName);
   const accessibility = findBlock(poi.blocks, 'accessibility');
   const internetAccess = findBlock(poi.blocks, 'internet_access');
   const delivery = findBlock(poi.blocks, 'delivery');
-  // const brewery = findBlock(poi.blocks, 'brewery');
-  // {brewery && <BreweryBlock block={brewery} />}
+  // const stars = findBlock(poi.blocks, 'stars');
+  // Example:
+  const stars = {
+    has_stars: 'yes', // yes or unknown
+    nb_stars: 4, // float
+  };
 
   if (!accessibility && !internetAccess && !hasActiveDeliveryModes(delivery)) {
```

Comment:
There will be the same problem with this block, we shouldn't render the whole "details" section if nothing is rendered inside, so we have to add a check there. And the condition is potentially complex (because the block can exist with a 'unknown' value), like for delivery it may require a dedicated function.